src/Tribe/Views/V2/View.php

```diff
@@ -1267,6 +1278,9 @@ protected function setup_repository_args( Context $context = null ) {
 			'view_override_offset' => true,
 		];
 
+		// Merge our global repository args with the primary event args.
+		$args = array_merge($args, $this->get_global_repository_args());
```

Comment:
```suggestion
		$args = array_merge( $args, $this->get_global_repository_args() );
```

The override order, where global repository arguments would override the View ones, seems odd; I think it should be the opposite, where global args are overridden by the View-specific ones.